src/server.h

```diff
@@ -2099,6 +2110,7 @@ struct redisCommand {
                    ACLs. A connection is able to execute a given command if
                    the user associated to the connection has this command
                    bit set in the bitmap of allowed commands. */
+    struct hdr_histogram* latency_histogram; /*points to the command latency command histogram (unit of time nanosecond) */
```

Comment:
remind me how much memory this consumes?
maybe we should free it when the config is disabled?

Reply:
ohh, i see the top comment describes memory usage..

Reply:
The total size per histogram should sit around 40 KiB. We only allocate those 40KiB when a command was called for the first time.

Reply:
so should we bother to free it if the config is disabled?
assume a normal user uses some 20-40 commands (that could be some 1.5mb).
i guess not a big price to pay (user will set the config and get that memory back on the next restart).
@madolson WDYT?